### PR TITLE
Rename Maintainers.md and GovernanceModel.md

### PR DESCRIPTION
**Pull request by andschwa**
To Committers.md and Governance.md for consistency.

This spun out of #2166.

I am assigning @openenclave/committee but we don't need a full consensus for this, I'm just renaming some files as previously discussed.

### CODE REVIEW

**Comment by andschwa** [comment]
I think it was the change to CODEOWNERS that pulled in the whole committers group as a code owner...which I'm fine with 😄

File: docs/CODEOWNERS [hunk 1]
```diff
@@ -46,7 +46,7 @@ CMakeLists.txt @andschwa @BRMcLaren @EmilAlexandruStoica
 /docs/DesignDocs/ @openenclave/committers
 
 # And governance documents.
-/docs/Contributing.md @openenclave/maintainers
-/docs/GovernanceModel.md @openenclave/maintainers
-/docs/Maintainers.md @openenclave/maintainers
-/docs/Releasing.md @openenclave/maintainers
```
Comment (dthaler) [inline docs/CODEOWNERS]: can we create a /process subdirectory under /docs to group these and just put openenclave/committee on the whole subdirectory?  We may have other docs over time that get added so want to reduce opportunities for errors.

Comment (andschwa) [reply docs/CODEOWNERS]: I don't think that'll work for Contributing.md as GitHub specifically references it from docs.

Comment (andschwa) [reply docs/CODEOWNERS]: (And then I think it would be weird to have them in different places.)

Comment (andschwa) [reply docs/CODEOWNERS]: In my opinion...not necessary 😉

**Comment by andschwa** [comment]
This "approved on behalf of" thing with GitHub is a little...weird.

**Pull request merged**